frontend/pages/Register/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import UserForm from '@shopgate/user/components/UserForm';
+import styles from './style';
+
+/**
+ * Register component
+ */
+class Register extends Component {
+  static contextTypes = {
+    i18n: PropTypes.func,
+  };
+
+  /**
+   * Returns the translated view title.
+   * @return {string}
+   */
+  get title() {
+    const { __ } = this.context.i18n();
+    return __('register.title');
+  }
+
+  /**
+   * @return {*}
+   */
+  render() {
+    // eslint-disable-next-line react/prop-types
+    const { View } = this.props;
+    return (
+      <View title={this.title}>
```

Comment:
In the design the title is actually empty in iOS and only the headline is shown underneath. 

This actually also applies to all pages in the addressbook (I will change that in a different ticket though I have to change there something anyway)